openerp/tools/config.py

```diff
@@ -529,14 +530,21 @@ def die(cond, msg):
             #if self.options['db_host']:
             #    self._generate_pgpassfile()
 
+        # server_wide_modules defaults to web,web_kanban if empty or unset
+        server_wide_modules = self.options['server_wide_modules'] = (
+            self.options['server_wide_modules']
+            if self.options['server_wide_modules']
+            else 'web,web_kanban'
+        )
+
```

Comment:
Yes @StefanRijnhart, doing it the way you mentioned is shorter, and I considered doing it.

But in the spirit of [The Zen of Python](https://www.python.org/dev/peps/pep-0020/), I think it's currently more readable and more idiomatic to use the '... if ... else ...` pattern than the Boolean short circuit.

But if you insist, I'll make the change.